Misc/NEWS.d/next/Library/2025-05-19-20-59-06.gh-issue-134209.anhTcF.rst

```diff
@@ -0,0 +1 @@
+The :meth:`curses.window.instr` and :meth:`curses.window.getstr` allocate its internal buffer on the heap instead of the stack. Increased its max buffer size to 2047 from of 1023, documenting the limit.
```

Comment:
```suggestion
:mod:`curses`: The :meth:`curses.window.instr` and :meth:`curses.window.getstr`
methods now allocate their internal buffer on the heap instead of the stack;
in addition, the max buffer size is increased from 1023 to 2047.
```